data/src/config/server.rs

```diff
@@ -65,6 +66,18 @@ pub struct Server {
     /// Commands which are executed once connected.
     #[serde(default)]
     pub on_connect: Vec<String>,
+    /// WHO poll interval for servers without away-notify.
+    #[serde(
+        default = "default_who_poll_interval",
+        deserialize_with = "deserialize_duration_from_u64"
+    )]
+    pub who_poll_interval: Duration,
+    /// WHO retry interval for servers without away-notify.
+    #[serde(
+        default = "default_who_retry_interval",
+        deserialize_with = "deserialize_duration_from_u64"
+    )]
+    pub who_retry_interval: Duration,
 }
```

Comment:
Make sure we implement Default for this struct vs derive so the durations can be defaulted properly when this entire struct is omitted / defaulted